storage/innobase/row/row0mysql.cc

```diff
@@ -1948,11 +1948,17 @@ or is tablespace .ibd file missing.
 @param[in] table                Table
 @param[in] trx                  Transaction
 @param[in] push_warning         true if we should push warning to user
+@param[in] err_on_tablespace_missing error that should be reported
+                                     when tablespace is missing, could
+                                     be either DB_TABLESPACE_NOT_FOUND
+                                     or DB_TABLE_NOT_FOUND
 @retval DB_IO_DECRYPT_FAIL    table is encrypted but decryption failed
 @retval DB_CORRUPTION           table is corrupted
-@retval DB_TABLESPACE_NOT_FOUND tablespace .ibd file not found */
-static dberr_t row_mysql_get_table_status(const dict_table_t *table, trx_t *trx,
-                                          bool push_warning = true) {
+@retval DB_TABLESPACE_NOT_FOUND tablespace .ibd file not found
+        or DB_TABLE_NOT_FOUND   depends on err_on_tablespace_missing */
```

Comment:
Split DB_TABLE_NOT_FOUND to its own retval

Reply:
done